### PR TITLE
impltented api for notes crud operations

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**
Hello, i have implemented  notes CRUD functionality using Api's . kindly review it and let me know what do think about it !
files to check -> NotesContext.js, services folder, validations folder, NotesPage folder, NotesCard.js
PREVIEW - https://blithe-la-nota.netlify.app/
PR - https://github.com/kgajriaAgarwal/blithe-lanota/pull/3

### CODE REVIEW

**Review by Naman-Saxena1 — APPROVED** [review]
Code quality looks good and preview functionality is also working fine 👍

**Review by mikesch803 — APPROVED** [review]
preview working great, nice ui, code is also good

File: src/Helpers/Context/NotesContext.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                    showErrorToast("Kindly login first !!")
+                }
+              }            
+          })
+          .catch((error) => {
+            showErrorToast("Unexpected error.Please try again later.")
+          })
+        }
+
+
```
Comment (Adarsh-Patel-Dev) [inline src/Helpers/Context/NotesContext.js:84]: remove extra space here

File: src/Pages/Shared/NoteCard/NoteCard.js [hunk 2]
```diff
@@ -1,77 +1,70 @@
...
     width: '98%',
-    display: 'flex',
     gap: '10px',
     margin: '10px',
   }));
 
-
 export const NoteCard = (props) =>{
 
+    console.log("props:",props )
```
Comment (Adarsh-Patel-Dev) [inline src/Pages/Shared/NoteCard/NoteCard.js:32]: remove console.log before merging

**Review by Adarsh-Patel-Dev — APPROVED** [review]
Overall code is well written
Suggested some minor changes 
You can merge after that

**Review by mehulsatardekar — APPROVED** [review]
everything looks good  and the functionality is also working good

**Pull request merged**